### PR TITLE
Patch 11

### PR DESCRIPTION
**Pull request by GAOXING-OK**
#### Hi! What is your PR about?

- [x] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

**Review by GAOXING-OK — COMMENTED** [review]
ok

**Review by GAOXING-OK — COMMENTED** [review]
ok

**Comment by GAOXING-OK** [comment]
ok

File: spaces/buger.eth/1 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (gokaiorg) [inline spaces/buger.eth/1:1]: Please delete this file.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Duplicate with #1181

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**